app/src/main/java/com/uncmorfi/data/repository/RepoNotify.kt

```diff
@@ -31,9 +38,16 @@ class RepoNotify(val context: Context) {
     }
 
     fun update(notification: Notification, id: Int = DEFAULT_ID) {
-        NotificationManagerCompat
-            .from(context)
-            .notify(id, notification)
+        if (ActivityCompat.checkSelfPermission(
+                context,
+                Manifest.permission.POST_NOTIFICATIONS
+            ) == PackageManager.PERMISSION_GRANTED
+        ) {
```

Comment:
Si tenés ganas, estaría bueno poner esto en una extesion fun en Helper.kt, tipo:

```suggestion
        if (context.isPermissionGranted(Manifest.permission.POST_NOTIFICATIONS)) {
```

Reply:
```
    @SuppressLint("MissingPermission")
    fun update(notification: Notification, id: Int = DEFAULT_ID) {
        if (context.isPermissionGranted(Manifest.permission.POST_NOTIFICATIONS)) {
            NotificationManagerCompat
                .from(context)
                .notify(id, notification)
            return
        }
```
En esta clase me tira un error

> Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission)

Lo suprimí con `@SuppressLint` ya que usa el mismo código que se llama dentro de la extension function.